src/thunks/cleanTrashedGroups.js

```diff
@@ -28,7 +28,7 @@ const removeGroupFromAllContacts = async (client, groupId) => {
   const contactQueryByGroupId = buildContactQueryByGroupId(groupId)
 
   const { data, next: hasMore } = await client.query(
-    contactQueryByGroupId.definition
+    contactQueryByGroupId.definition()
```

Comment:
nit: je ne crois pas qu'il y ait de consensus sur le sujet mais si pas d'arg, pas nécessaire que ce soit une fonction. Tu peux laisser en l'état, c'est juste un nit pour planter une graine :p

Reply:
Old query used into `<Query>` seems to need a callback function for definition. To harmonise the query file I used a callback function on all queries. cc @Ldoppea